### PR TITLE
[docs] update linux/win installation guide

### PR DESCRIPTION
**Pull request by Jacob27**
### Details:
 - *update linux installation guide*
 - *update windows installation guide*

### Tickets:
 - *CVS-71745*
 - *CVS-71748*

### CODE REVIEW

**Comment by Jacob27** [comment]
http://openvino-docs.inn.intel.com/jacek-linux-win-installation-guide-1/openvino_docs_install_guides_installing_openvino_windows.html

File: docs/install_guides/installing-openvino-linux.md [hunk 1]
```diff
@@ -153,184 +130,118 @@ These dependencies are required for:
...
-   ```sh
-   source /opt/intel/openvino_2022/bin/setupvars.sh
-   ```
+> **NOTE**: You can also run this script every time when you start new terminal session. Open `~/.bashrc` in your favorite editor, and add `source <INSTALL_DIR>/intel/openvino_2022/bin/setupvars.sh`. Next time when you open a terminal, you will see `[setupvars.sh] OpenVINO environment initialized`. Changing `.bashrc` is not recommended when you have many OpenVINO versions on your machine and want to switch among them, as each may require different setup.
 
-4. Save and close the file: press the **Esc** key and type `:wq`.
+The environment variables are set. Next, you can download some additional tools.
 
-5. To verify the change, open a new terminal. You will see `[setupvars.sh] OpenVINO environment initialized`.
+## <a name="model-optimizer">Step 4 (Optional): Download additional components
```
Comment (slyubimt) [inline docs/install_guides/installing-openvino-linux.md:145]: Can we add here information on how to download OpenCV using a script supplied by installer? 
+ @mshabunin to provide instructions.

File: docs/install_guides/installing-openvino-windows.md [hunk 2]
```diff
@@ -1,250 +1,150 @@
...
-
-3. Right-click the adapter name and select **Properties**.
+> **NOTE**: If you see an error indicating Python is not installed when you know you installed it, your computer might not be able to find the program. Check your system environment variables, and add Python if necessary.
 
-4. Click the **Driver** tab to see the driver version. 
-   ![](../img/DeviceDriverVersion.PNG)
+The environment variables are set. Next, you can download some additional tools.
 
-You are done updating your device driver and are ready to use your GPU. Proceed to the <a href="#get-started">Start Using the Toolkit</a> section to learn the basic OpenVINO™ toolkit workflow and run code samples and demo applications.
+## <a name="model-optimizer">Step 3 (Optional): Download additional components
```
Comment (slyubimt) [inline docs/install_guides/installing-openvino-windows.md:98]: Can we add here information on how to download OpenCV using a script supplied by Windows installer?

@mshabunin to provide instructions.

File: docs/install_guides/installing-openvino-linux.md [hunk 3]
```diff
@@ -1,52 +1,40 @@
...
-| Additional Tools | A set of tools to work with your models including [Accuracy Checker utility](@ref omz_tools_accuracy_checker), [Post-Training Optimization Tool](@ref pot_README), [Model Downloader](@ref omz_tools_downloader) and others |
-| [Documentation for Pre-Trained Models ](@ref omz_models_group_intel) | Documentation for the pre-trained models available in the [Open Model Zoo repo](https://github.com/openvinotoolkit/open_model_zoo). |
-| Deep Learning Streamer (DL Streamer) | Streaming analytics framework, based on GStreamer, for constructing graphs of media analytics components. For the DL Streamer documentation, see [DL Streamer Samples](@ref gst_samples_README), [API Reference](https://openvinotoolkit.github.io/dlstreamer_gst/), [Elements](https://github.com/openvinotoolkit/dlstreamer_gst/wiki/Elements), [Tutorial](https://github.com/openvinotoolkit/dlstreamer_gst/wiki/DL-Streamer-Tutorial). |
+@sphinxdirective
+.. tab:: Operating Systems
 
-## System Requirements
+  * Ubuntu 18.04.x long-term support (LTS), 64-bit
+  * Ubuntu 20.04.0 long-term support (LTS), 64-bit
+  * Red Hat* Enterprise Linux* 8.2, 64-bit
```
Comment (slyubimt) [inline docs/install_guides/installing-openvino-linux.md:12]: In general RHEL 8.x supported
```suggestion
  * Red Hat* Enterprise Linux* 8.x, 64-bit
```

File: docs/install_guides/installing-openvino-linux.md [hunk 4]
```diff
@@ -1,52 +1,40 @@
...
-| Additional Tools | A set of tools to work with your models including [Accuracy Checker utility](@ref omz_tools_accuracy_checker), [Post-Training Optimization Tool](@ref pot_README), [Model Downloader](@ref omz_tools_downloader) and others |
-| [Documentation for Pre-Trained Models ](@ref omz_models_group_intel) | Documentation for the pre-trained models available in the [Open Model Zoo repo](https://github.com/openvinotoolkit/open_model_zoo). |
-| Deep Learning Streamer (DL Streamer) | Streaming analytics framework, based on GStreamer, for constructing graphs of media analytics components. For the DL Streamer documentation, see [DL Streamer Samples](@ref gst_samples_README), [API Reference](https://openvinotoolkit.github.io/dlstreamer_gst/), [Elements](https://github.com/openvinotoolkit/dlstreamer_gst/wiki/Elements), [Tutorial](https://github.com/openvinotoolkit/dlstreamer_gst/wiki/DL-Streamer-Tutorial). |
+@sphinxdirective
+.. tab:: Operating Systems
 
-## System Requirements
+  * Ubuntu 18.04.x long-term support (LTS), 64-bit
+  * Ubuntu 20.04.0 long-term support (LTS), 64-bit
```
Comment (slyubimt) [inline docs/install_guides/installing-openvino-linux.md:11]: Align a format with previous line:
```suggestion
  * Ubuntu 20.04.x long-term support (LTS), 64-bit
```

**Review by slyubimt — CHANGES_REQUESTED** [review]

**Review by slyubimt — APPROVED** [review]

**Comment by Jacob27** [comment]
http://openvino-docs.inn.intel.com/jacek-linux-win-installation-guide-1/openvino_docs_install_guides_installing_openvino_windows.html

**Review by andrew-zaytsev — CHANGES_REQUESTED** [review]

**Review by andrew-zaytsev — CHANGES_REQUESTED** [review]

File: docs/install_guides/installing-openvino-linux-header.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+   Installer <openvino_docs_install_guides_installing_openvino_linux>
+   APT <openvino_docs_install_guides_installing_openvino_apt>
+   YUM <openvino_docs_install_guides_installing_openvino_yum>
+   Docker <openvino_docs_install_guides_installing_openvino_docker_linux>
+
+@endsphinxdirective
+
+If you want to install OpenVINO™ on your Linux machine, there are a few ways to accomplish this. We prepared following options for you: 
+
+* [Install OpenVINO™ with an Installer](installing-openvino-linux.md)
```
Comment (andrew-zaytsev) [inline docs/install_guides/installing-openvino-linux-header.md:18]: OpenVINO™ --> Intel® Distribution of OpenVINO™ toolkit

File: docs/install_guides/installing-openvino-linux.md [hunk 6]
```diff
@@ -1,52 +1,40 @@
 # Install and Configure Intel® Distribution of OpenVINO™ toolkit for Linux* {#openvino_docs_install_guides_installing_openvino_linux}
```
Comment (andrew-zaytsev) [inline docs/install_guides/installing-openvino-linux.md:1]: Let's remove asterisks for all third-party product names (it was approved by Marketing): Windows, Linux, Ubuntu etc

**Review by andrew-zaytsev — CHANGES_REQUESTED** [review]

**Review by andrew-zaytsev — APPROVED** [review]

File: docs/install_guides/installing-openvino-windows-ivad-vpu.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+.. _vpu guide windows:
+
+@endsphinxdirective
+
+
+To enable inference on Intel® Vision Accelerator Design with Intel® Movidius™ VPUs, the following additional installation steps are required:
+
+  1. Download and install <a href="https://www.microsoft.com/en-us/download/details.aspx?id=48145">Visual C++ Redistributable for Visual Studio 2017</a>
+  2. Check with a support engineer if your Intel® Vision Accelerator Design with Intel® Movidius™ VPUs card requires SMBUS connection to PCIe slot (most unlikely). Install the SMBUS driver only if confirmed (by default, it's not required):
+      1. Go to the `<INSTALL_DIR>\deployment_tools\inference-engine\external\hddl\drivers\SMBusDriver` directory, where `<INSTALL_DIR>` is the directory in which the Intel Distribution of OpenVINO™ toolkit is installed.
```
Comment (slyubimt) [inline docs/install_guides/installing-openvino-windows-ivad-vpu.md:14]: This path is outdated

**Pull request merged**